reference/5.1/Microsoft.PowerShell.Core/About/about_PSModulePath.md

```diff
@@ -0,0 +1,181 @@
+---
+keywords: powershell,cmdlet
+locale: en-us
+ms.date: 04/13/2020
+online version: https://docs.microsoft.com/powershell/module/microsoft.powershell.core/about/about_PSModulePath?view=powershell-5.1&WT.mc_id=ps-gethelp
+schema: 2.0.0
+title: about_PSModulePath
+---
+# About PSModulePath
+
+The `$env:PSModulePath` environment variable contains a list of folder
+locations that are searched to find modules and resources.
+
+By default, the effective locations assigned to `$env:PSModulePath` are:
+
+- System-wide locations: These folders contain modules that ship with
+  PowerShell. These modules are store in the `$PSHOME\Modules` folder. This is
+  also the location where the Windows management modules are installed.
+
+- User-installed modules: These are modules installed by the user.
+  `Install-Module` has a **Scope** parameter that allows you to specify
+  whether the module is installed for the current user or for all users. For
+  more information, see
+  [Install-Module](../../PowerShellGet/Install-Module.md).
+
+  - On Windows, the location of the user-specific **CurrentUser** scope is
+    the `$HOME\Documents\PowerShell\Modules` folder. The location of the
+    **AllUsers** scope is `$env:ProgramFiles\PowerShell\Modules`.
+  - On non-Windows systems, the location of the user-specific **CurrentUser**
+    scope is the `$HOME/.local/share/powershell/Modules` folder. The location
+    of the **AllUsers** scope is `/usr/local/share/powershell/Modules`.
+
+In addition, setup programs that install modules in other directories, such
+as the Program Files directory, can append their locations to the value of
+`$env:PSModulePath`.
+
+> [!NOTE]
+> On Windows, the user-specific location is the `PowerShell\Modules` folder
+> located in the **Documents** folder in your user profile. The specific path
+> of that location varies by version of Windows and whether or not you are
+> using folder redirection. Microsoft OneDrive can also change the location of
+> your **Documents** folder. You can verify the location of your **Documents**
+> folder using the following command:
+> `[Environment]::GetFolderPath('MyDocuments')`.
+
+## Modifying PSModulePath
+
+To change the default module directories for the current session, use the
+following command format to change the value of the `PSModulePath`
+environment variable.
+
+For example, to add the `C:\Program Files\Fabrikam\Modules` directory to
+the value of the PSModulePath environment variable, type:
+
+```powershell
+$Env:PSModulePath = $Env:PSModulePath+";C:\Program Files\Fabrikam\Modules"
+```
+
+The semi-colon (`;`) in the command separates the new path from the path that
+precedes it in the list. On non-Windows platforms, the colon (`:`) separates
```

Comment:
Do you want to introduce the use of [system.io.path]::PathSeparator for cross platform scripts?

Reply:
Might also be worth sharing a trick I use making it easier to read psmodulepath:

```powershell
$env:PSModulePath.Split([system.io.path]::PathSeparator)
```